### PR TITLE
Use nhost dashboard

### PR DESCRIPTION
**Pull request by elephant3**
## Description
Plug [nhost dashboard](https://github.com/nhost/nhost/tree/main/dashboard) into CLI

### CODE REVIEW

**Comment by elitan** [comment]
Since we plan to make the Nhost Dashboard the default UI (instead of Hasura Console), does it make sense to remove or rename this flag?

Random idea: Maybe `--ui=nhost` and `--ui=hasura` is default?

**Comment by elephant3** [comment]
> Since we plan to make the Nhost Dashboard the default UI (instead of Hasura Console), does it make sense to remove or rename this flag?
> 
> Once the Nhost Dashboard is the default UI this `--dashboard` flag wouldn't make much sense.
> 
> Random idea: Maybe `--ui=nhost` and `--ui=hasura` is default?

Sounds good

**Comment by elitan** [comment]
@nunopato What do you think?

**Comment by elephant3** [comment]
Updated, now `--ui nhost` shall be provided for our dashboard

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ It's recommended to use the Nhost CLI and the [Nhost GitHub Integration](https:/
 
 ## Services:
 
+- [Dashboard](https://github.com/nhost/nhost/tree/main/dashboard)
```
Comment (elitan) [inline README.md:22]:
```suggestion
- [Nhost Dashboard](https://github.com/nhost/nhost/tree/main/dashboard)
```

File: README.md [hunk 2]
```diff
@@ -53,6 +54,12 @@ Start the development environment:
 nhost up
 ```
 
+You can also use our new dashboard:
```
Comment (elitan) [inline README.md:57]:
```suggestion
You can also use the Nhost Dashboard:
```

File: nhost/service/endpoints.go [hunk 3]
```diff
@@ -37,11 +39,12 @@ URLs:
 - Storage:		%s
 - Functions:		%s
 
+- Dashboard:		%s
```
Comment (elitan) [inline nhost/service/endpoints.go:42]:
```suggestion
- Nhost Dashboard:		%s
```

File: nhost/service/endpoints.go [hunk 4]
```diff
@@ -37,11 +39,12 @@ URLs:
...
 - Hasura console:	%s
```
Comment (elitan) [inline nhost/service/endpoints.go:43]:
```suggestion
- Hasura Console:	%s
```

**Review by elitan — CHANGES_REQUESTED** [review]
Small copy updates.

**Review by elitan — APPROVED** [review]

**Pull request merged**